base_layer/core/src/base_node/comms_interface/inbound_handlers.rs

```diff
@@ -407,6 +407,15 @@ where T: BlockchainBackend + 'static
     ) -> Result<(), CommsInterfaceError> {
         let NewBlock { block_hash } = new_block;
 
+        if self.blockchain_db.inner().is_add_block_locked() {
```

Comment:
nit: I prefer a name like `prevent_add_block` or `disable_add_block`

```suggestion
        if self.blockchain_db.inner().is_add_block_locked() {
```

Reply:
Cool, I like `disable` better. As a consistency thing, I name functions that return a bool starting with `is_x` will rename to `is_add_block_disabled` and rename everything else along those lines